pom.xml

```diff
@@ -61,17 +61,18 @@
   </developers>
 
   <scm>
-    <connection>scm:git:https://github.com/jenkinsci/thin-backup-plugin.git</connection>
-    <developerConnection>scm:git:https://git@github.com/jenkinsci/thin-backup-plugin.git</developerConnection>
+    <connection>scm:git:https://github.com/${gitHubRepo}.git</connection>
+    <developerConnection>scm:git:git@github.com:${gitHubRepo}.git</developerConnection>
     <tag>${scmTag}</tag>
-    <url>https://github.com/jenkinsci/thin-backup-plugin</url>
+    <url>https://github.com/${gitHubRepo}</url>
   </scm>
 
   <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <revision>1.19</revision>
+    <changelist>-SNAPSHOT</changelist>
     <jenkins.version>2.401.3</jenkins.version>
+    <gitHubRepo>jenkinsci/thin-backup-plugin</gitHubRepo>
```

Comment:
Enables [incremental builds](https://github.com/jenkinsci/incrementals-tools) in CI.